llvm/examples/BrainF/BrainF.cpp

```diff
@@ -149,8 +149,7 @@ void BrainF::header(LLVMContext& C) {
 
     //declare i32 @puts(i8 *)
```

Comment:
As a followup, this code could use same larger cleanup to use opaque pointers in the comments and remove some unnecessary stuff like the zero-offset GEP below.